koku/masu/database/trino_sql/reporting_gcpcostentrylineitem_daily_summary.sql

```diff
@@ -61,9 +61,6 @@ FROM hive.{{schema | sqlsafe}}.{{table | sqlsafe}}
 CROSS JOIN
     cte_pg_enabled_keys as pek
 WHERE source = '{{source_uuid | sqlsafe}}'
-    AND year = '{{year | sqlsafe}}'
-    AND month = '{{month | sqlsafe}}'
-    AND invoice_month = '{{year | sqlsafe}}{{month | sqlsafe}}'
```

Comment:
Why are we removing these from the SQL here? `year` and `month` are partitions on the data and should not be removed even if we remove `invoice_month`

Reply:
This might not be accurate and obviously needs testing, I just chucked this together quickly. :smile:  That said since we are not using invoice month I think we want to select and insert the data from both partitions at the same time as we no longer trigger this per month. The selected and inserted data still contains the invoice month so it _should_ still land in the correct places, but I could still be overlooking/missing something.